media-video/pipewire/pipewire-0.3.51-r1.ebuild

```diff
@@ -248,12 +254,26 @@ multilib_src_install_all() {
 		dosym ../../../usr/share/alsa/alsa.conf.d/99-pipewire-default-hook.conf /etc/alsa/conf.d/99-pipewire-default-hook.conf
 	fi
 
+	# Enable required wireplumber alsa and bluez monitors
+	if use sound-server; then
```

Comment:
In dealing with the bigger issues I forgot that according to Gentoo QA rules such use flags that only operate on data files and dependencies but not the build system are not allowed. PulseAudio has a a long fought exception and I'd be very happy to not see such exceptions propagate into more or new packages.